exporter/stackdriverexporter/config.go

```diff
@@ -33,6 +34,11 @@ type Config struct {
 	// Timeout for all API calls. If not set, defaults to 12 seconds.
 	exporterhelper.TimeoutSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 	ResourceMappings               []ResourceMapping        `mapstructure:"resource_mappings"`
+	// ClientOptions are additional options to be passed
+	// to the underlying Cloud Monitoring API client.
+	// Must be set programmatically (no support via declarative config).
+	// Optional.
+	ClientOptions []option.ClientOption
```

Comment:
Optional: I wonder if it would be cleaner to put this into a separate struct that is passed into the `NewFactory()` function explicitly instead of polluting the `Config` struct?

Reply:
The problem with making it part of `NewFactory` is that it will be "static", i.e. will apply to all instances of exporter.
One could imagine configuring exporters with different client options.